### ISSUE TITLE
TTL parameter is required for registering script Consul health check

### ISSUE THREAD

**Issue by mbarouski** [bitsofinfo/hazelcast-consul-discovery-spi#31]
I'm trying to register script health check with such config:
```
        String discoveryHostname = "localhost";
        int discoveryPort = 8500;
        String hazelcastAddress = "localhost";
        String hazelcastPassword = "password";

        Config config = new Config();

        config.getProperties().setProperty("hazelcast.discovery.enabled", "true");

        config.getNetworkConfig().getJoin().getMulticastConfig().setEnabled(false);
        config.getNetworkConfig().getJoin().getAwsConfig().setEnabled(false);
        config.getNetworkConfig().getJoin().getTcpIpConfig().setEnabled(false);

        config.getNetworkConfig().setPortAutoIncrement(true);
        config.getNetworkConfig().setPort(port);

        Map<String, Comparable> properties = new HashMap<>();

        properties.put("consul-host", discoveryHostname);
        properties.put("consul-port", String.valueOf(discoveryPort));
        properties.put("consul-service-name", "hazelcast");
        properties.put("consul-healthy-only", "true");
        properties.put("consul-service-tags", "hazelcast");
        properties.put("consul-discovery-delay-ms", "0");
        properties.put("consul-registrator", LocalDiscoveryNodeRegistrator.class.getName());

        properties.put("consul-registrator-config", format("{\n" +
          "\"preferPublicAddress\": false,\n" +
          "\"healthCheckProvider\": \"%s\",\n" +
          "\"healthCheckScript\": \"nc -z %s %s\",\n" +
          "\"healthCheckScriptIntervalSeconds\": 30\n" +
          "}", ScriptHealthCheckBuilder.class.getName(), hazelcastAddress, port));

        DiscoveryStrategyConfig strategy = new DiscoveryStrategyConfig(ConsulDiscoveryStrategy.class.getName(), properties);
     
   config.getNetworkConfig().getJoin().getDiscoveryConfig().addDiscoveryStrategyConfig(strategy);

        return config;
```
and get error:
```
Caused by: com.orbitz.consul.ConsulException: Consul request failed with status [400]: Invalid check: TTL must be > 0 for TTL checks
```.

**Comment by bitsofinfo** [comment]
I have not had time to look at this, likely an easy fix, if you can prep a patch please submit it as a PR and I'll review and get it out. Thanks

**Comment by mbarouski** [comment]
Will try to find time to do it.
But I don't need this feature now because I used `http health check` which is available since `Hazelcast 3.9`.

**Comment by mbarouski** [comment]
@bitsofinfo My issue was resolved in version `1.0-RC9` but I used `1.0-RC8`. 
Since this the issue can be closed.
